src/misc/setup.cpp

```diff
@@ -983,18 +983,26 @@ const Section_line &Config::GetOverwrittenAutoexecSection() const
 
 bool Config::ParseConfigFile(const std::string &type, const std::string &configfilename)
 {
+	std::error_code ec;
+	const std_fs::path cfg_path = configfilename;
+	const auto conanical_path = std_fs::canonical(cfg_path, ec);
+	if (ec)
+		return false;
+	
+	if (contains(configFilesCanonical, conanical_path)) {
+		LOG_INFO("CONFIG: Skipping duplicate config file '%s'", 
+			configfilename.c_str());
+		return true;
+	}
 	// static bool first_configfile = true;
-	ifstream in(configfilename);
-	if (!in)
+	ifstream in(conanical_path);
```

Comment:
`Conanical` (small typo)
 🌴

Reply:
Sigh... I always get that one wrong. Thanks.